packages/smooth_app/lib/pages/user_management/login_page.dart

```diff
@@ -91,151 +89,163 @@ class _LoginPageState extends State<LoginPage> {
           alignment: Alignment.topCenter,
           child: SizedBox(
             width: size.width * 0.7,
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.center,
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: <Widget>[
-                const Spacer(flex: 4),
-
-                Text(
-                  appLocalizations.sign_in_text,
-                  textAlign: TextAlign.center,
-                  style: theme.textTheme.headline1?.copyWith(
-                    fontSize: 20.0,
-                    fontWeight: FontWeight.w700,
-                    color: theme.colorScheme.onSurface,
+            child: AutofillGroup(
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.center,
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  const Spacer(flex: 4),
+
+                  Text(
+                    appLocalizations.sign_in_text,
+                    textAlign: TextAlign.center,
+                    style: theme.textTheme.headline1?.copyWith(
+                      fontSize: 20.0,
+                      fontWeight: FontWeight.w700,
+                      color: theme.colorScheme.onSurface,
+                    ),
                   ),
-                ),
 
-                const Spacer(flex: 8),
+                  const Spacer(flex: 8),
 
-                if (_wrongCredentials) ...<Widget>[
-                  SmoothCard(
-                    padding: const EdgeInsets.all(10.0),
-                    color: Colors.red,
-                    child: Text(appLocalizations.incorrect_credentials),
+                  if (_wrongCredentials) ...<Widget>[
+                    SmoothCard(
+                      padding: const EdgeInsets.all(10.0),
+                      color: Colors.red,
+                      child: Text(appLocalizations.incorrect_credentials),
+                    ),
+                    const Spacer(
+                      flex: 1,
+                    )
+                  ],
+
+                  //Login
+                  SmoothTextFormField(
+                    type: TextFieldTypes.PLAIN_TEXT,
+                    controller: userIdController,
+                    hintText: appLocalizations.login,
+                    textColor: customGrey,
+                    backgroundColor: textFieldBackgroundColor,
+                    prefixIcon: const Icon(Icons.person),
+                    enabled: !_runningQuery,
+                    // Moves focus to the next field
+                    textInputAction: TextInputAction.next,
+                    autofillHints: const <String>[
+                      AutofillHints.username,
+                      AutofillHints.name,
```

Comment:
why name?

Reply:
I thought maybe some people signed up with their real name but I just checked, the username doesn't allow punctuation. Already working further on this code, fixing this in a follow up PR